dbms/CMakeLists.txt

```diff
@@ -173,6 +173,7 @@ target_link_libraries (clickhouse_common_io
         PRIVATE
     apple_rt
     ${CMAKE_DL_LIBS}
+    ${HDFS3_LIBRARY}
```

Comment:
Missed `target_include_directories` for HDFS.

Reply:
Do you update submodules? libhdfs3 will be in submodule.

Reply:
Yes, I have updated submodules, but compiler didn't see `hdfs.h` header without inserting `target_include_directories(dbms ${HDFS_INCLUDE_DIR})` in CMakeLists.txt:
```
[729/1377] Building CXX object dbms/CMakeFiles/dbms.dir/src/Interpreters/InterpreterInsertQuery.cpp.o
FAILED: /usr/bin/ccache /usr/bin/clang++-6.0   -DBOOST_SYSTEM_NO_DEPRECATED -DPOCO_STATIC -DPOCO_UNBUNDLED_ZLIB -DUNALIGNED_OK -DWITH_GZFILEOP -DX86_64 -DZLIB_COMPAT -isystem contrib/jemalloc-cmake/include -isystem contrib/jemalloc-cmake/include_linux_x86_64 -isystem contrib/libsparsehash -isystem contrib/libdivide -isystem contrib/poco/Data/include -Idbms/src -isystem contrib/dou
ble-conversion -isystem contrib/libpcg-random/include -Ilibs/libcommon/include -isystem contrib/poco/Foundation/include -Icontrib/zlib-ng -Icontrib/cityhash102/include -isystem contrib/boost -isystem contrib/poco/Net/include -isystem contrib/poco/Util/include -isystem contrib/poco/XML/include -isystem contrib/poco/JSON/include -isystem contrib/re2_st -isystem contrib/re2 -Ilibs/li
bpocoext/include -Ilibs/libmysqlxx/include -Icontrib/mariadb-connector-c-cmake/linux_x86_64/include -Icontrib/mariadb-connector-c-cmake/common/include -Icontrib/mariadb-connector-c/include -isystem contrib/ssl/include -Icontrib/libbtrie/include -isystem contrib/poco/Data/ODBC/include -Icontrib/unixodbc-cmake/linux_x86_64 -Icontrib/unixodbc/include -Icontrib/unixodbc/libltdl -Icont
rib/unixodbc/libltdl/libltdl -isystem contrib/poco/MongoDB/include -isystem contrib/poco/NetSSL_OpenSSL/include -isystem contrib/poco/Crypto/include -isystem contrib/librdkafka/src -fdiagnostics-color=always -std=c++1z  -pipe -msse4.1 -msse4.2 -mpopcnt  -fno-omit-frame-pointer  -Wall -Wno-unused-command-line-argument  -Wnon-virtual-dtor -Wextra -Wextra-semi -Wcomma -Winconsistent-
missing-destructor-override -Wunused-exception-parameter -Wshadow-uncaptured-local -Wredundant-parens -Wzero-as-null-pointer-constant -O2 -g -DNDEBUG -O3    -pthread -MMD -MT dbms/CMakeFiles/dbms.dir/src/Interpreters/InterpreterInsertQuery.cpp.o -MF dbms/CMakeFiles/dbms.dir/src/Interpreters/InterpreterInsertQuery.cpp.o.d -o dbms/CMakeFiles/dbms.dir/src/Interpreters/InterpreterInse
rtQuery.cpp.o -c dbms/src/Interpreters/InterpreterInsertQuery.cpp
In file included from dbms/src/Interpreters/InterpreterInsertQuery.cpp:27:
dbms/src/IO/ReadBufferFromHDFS.h:4:10: fatal error: 'hdfs/hdfs.h' file not found
#include <hdfs/hdfs.h>
         ^~~~~~~~~~~~~
1 error generated.
[729/1377] Building CXX object dbms/CMakeFiles/dbms.dir/src/Interpreters/Join.cpp.o
[729/1377] Building CXX object dbms/CMakeFiles/dbms.dir/src/Interpreters/Aggregator.cpp.o
ninja: build stopped: subcommand failed.
```

Reply:
I just delivered a fix for this